packages/jsonc/jsonc.test.mjs

```diff
@@ -0,0 +1,72 @@
+import assert from 'node:assert/strict';
+import { execPath } from 'node:process';
+import { describe, it } from 'node:test';
+import { fileURLToPath } from 'node:url';
+
+import { spawnPromisified } from '../../utils/spawn-promisified.mjs';
+
+const TEST_CASE = ['json', 'jsonc'];
+
+describe('jsonc (e2e)', () => {
+	for (const type of TEST_CASE) {
```

Comment:
This may be causing problems. It makes the test non-statically analysable. I think there's some trick you have to do to get this to work :(

Reply:
so just write long test suite ?

Reply:
I think you may have to use `async` + `await test` (instead of `it`)